### ISSUE TITLE
Add permalinks to documentation headings

### ISSUE THREAD

**Issue by RazerM** [getlogbook/logbook#197]
The documentation doesn't have permalinks on hover, like sphinx does by default, so I can't link to a specific section of the documentation.

This can be enabled with the following patch:

```diff
diff --git a/docs/conf.py b/docs/conf.py
index ce3bcd5..aa90731 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -160,7 +160,7 @@ html_static_path = ['_static']
 # If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
 # html_show_copyright = True
 
-html_add_permalinks = ''
+# html_add_permalinks = ''
 
 # If true, an OpenSearch description file will be output, and all pages will
 # contain a <link> tag referring to it.  The value of this option must be the
diff --git a/docs/sheet/static/sheet.css_t b/docs/sheet/static/sheet.css_t
index e0c3734..7d518ec 100644
--- a/docs/sheet/static/sheet.css_t
+++ b/docs/sheet/static/sheet.css_t
@@ -143,3 +143,9 @@ p.output-caption {
     font-size: small;
     margin: 0px;
 }
+
+a.headerlink {
+    font-size: 0.5em;
+    text-decoration: none;
+    padding-left: 0.2em;
+}
```

I didn't open a pull request since I don't know if this is the correct CSS to do this, but it at least looks correct.

**Comment by vmalloc** [comment]
It seems like the right way to do it. Feel free to open it as a PR.

**Comment by RazerM** [comment]
Resolved by 6cc6017ed573da4be5e1d375a5519fb028a30929
